cmd/airflow.go

```diff
@@ -312,8 +317,18 @@ func airflowInit(cmd *cobra.Command, _ []string, out io.Writer) error {
 		useAstronomerCertified = true
 	}
 
+	// Validate runtimeVersion and airflowVersion
+	if airflowVersion != "" && runtimeVersion != "" {
+		return errInvalidBothAirflowAndRuntimeVersions
+	}
+	if useAstronomerCertified && runtimeVersion != "" {
+		fmt.Println("You provided a runtime version with the --use-astronomer-certified flag. Thus, this command will ignore the --runtime-version value you provided.")
+		runtimeVersion = ""
+	}
+
+	// if runtime version is set, use runtime version. If user specifies --use-astronomer-certified, or use runtime but didn't provide version, get default image tag
```

Comment:
@AmFlint if you run go test with coverage you can see in html report:

```
go tool cover -html=cover.out
go test -coverprofile cover.out -v -tags=containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper ./...
```

<img width="1393" alt="image" src="https://user-images.githubusercontent.com/1374633/165711112-3bbb7ceb-e427-42a2-8cbd-a17124dc22b2.png">